vitest.config.ts

```diff
@@ -1,14 +1,15 @@
 import tsconfigPaths from 'vite-tsconfig-paths'
 import { defineConfig } from 'vitest/config'
+// @ts-ignore: No types available
+import { lezer } from '@lezer/generator/rollup'
```

Comment:
Don't @ me. This is real production code we have in the other configuration files and the package that does the grammar import. None of it is typed. I need this to make `src/components/ProjectSidebarMenu.spec.tsx` work with the new project filtering feature in vitest. 


Have absolutely no idea how this worked previously.